migrations/20200304211821-unique-accountId.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface) => {
+        return queryInterface.addIndex(
```

Comment:
I think we might have problem with this migration as our current DB doesn't have unique account IDs. So we need some more complicated strategy (e.g. keep only latest entry for given accountId).

Reply:
@chadoh I think it's best to take snapshot of current accounts DB from Render and test with it locally, to make sure all migrations apply cleanly.

Reply:
Good call! Yes, I will test it out locally and modify this migration to work with testnet data before merging.

Reply:
I added a new migration that runs before this one which deletes all but the most recent of each set of duplicates. Using yesterday's dump of production data, here's what running migrations looks like for me locally:

```
 contract-helper ⮀ recoveryMethods ⮀ psql accounts_development helper -c 'select count(*) from "Accounts";'
 count
-------
  2226
(1 row)

 contract-helper ⮀ recoveryMethods ⮀ sequelize db:migrate

Sequelize CLI [Node: 12.14.1, CLI: 5.5.1, ORM: 5.21.5]

Loaded configuration file "config/config.js".
Using environment "development".
== 20200304211820-drop-duplicate-accountIds: migrating =======
== 20200304211820-drop-duplicate-accountIds: migrated (0.067s)

== 20200304211821-unique-accountId: migrating =======
== 20200304211821-unique-accountId: migrated (0.045s)

== 20200325154731-add-recovery-methods: migrating =======
== 20200325154731-add-recovery-methods: migrated (0.016s)

== 20200325204048-constrain-recoverymethods: migrating =======
== 20200325204048-constrain-recoverymethods: migrated (0.043s)

== 20200325204238-mv-data-from-accounts-to-recoverymethods: migrating =======
== 20200325204238-mv-data-from-accounts-to-recoverymethods: migrated (0.231s)

== 20200325213728-drop-phone-and-email-from-accounts: migrating =======
== 20200325213728-drop-phone-and-email-from-accounts: migrated (0.008s)

 contract-helper ⮀ recoveryMethods ⮀ psql accounts_development helper -c 'select count(*) from "Accounts";'
 count
-------
  2138
(1 row)
```